en_US/changes/changes-ee-v5.md

```diff
@@ -1,5 +1,203 @@
+
+  **Current limitations**:
+
+  - Only [S3Tables](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-tables.html) catalogs are supported (hence table data and metadata must live in S3).
+  - Only [Iceberg table format version 2](https://iceberg.apache.org/spec/#version-2-row-level-deletes) is supported.
+  - Only the following partition transform functions are supported:
+    - `identity`
+    - `void`
+    - `bucket[N]`
+  - Data files are written only in [Avro](https://avro.apache.org/docs/1.12.0/specification/).
+
+#### Multi-Tenancy
+
+- [#15253](https://github.com/emqx/emqx/pull/15253) Added two new multi-tenancy APIs: `GET /mt/ns_list_details` and `GET /mt/ns_list_managed_details`.  Both work similarly to their existing counterpars, but returns extra metadata associated with the namespace besides its name.
+- [#15160](https://github.com/emqx/emqx/pull/15160) Added the `DELETE /mt/bulk_delete_ns` API for multi-tenancy management, which allows deleting namespaces in bulk.
+
+#### CLI
+
+- [#15158](https://github.com/emqx/emqx/pull/15158) Added new `emqx ctl conf remove x.y.z` command, which removes the configuration key path `x.y.z` from the existing configuration.
+
+#### Gateway
+
+- [#15138](https://github.com/emqx/emqx/pull/15138) Introduced NATS Gateway for accepting NATS client connections over TCP/TLS, WS/WSS transport protocols.
+
+  For example, the NATS gateway will transform the following NATS message into an MQTT message with the topic `sub/t` and payload `hello`, while supporting seamless integration with existing EMQX features such as the rule engine, data integration, and more:
+  ```
+  PUB sub.t 5  
+  hello
+  ```
+
+
+#### Durable Storage
+
+- [#15043](https://github.com/emqx/emqx/pull/15043) Instrument the DS Raft backend with basic metrics to provide insights into cluster status, database overview, shard replication, and replica transitions.
+
+### Bug Fixes
+
+#### Access Control
+
+- [#15184](https://github.com/emqx/emqx/pull/15184) Fixed incorrect error message format when blacklist creation fails.
+
+#### Clustering
+
+- [#15304](https://github.com/emqx/emqx/pull/15304) Fixed the problem related to core node discovery by replicant nodes when using `static` discovery strategy.
+
+  Previously, the replicants could ignore core nodes not explicitly listed in the `static_seeds` list.
+  This could lead to an inconsistent cluster view and load imbalance.
+
+- [#15180](https://github.com/emqx/emqx/pull/15180)Fixed an issue in `ekka_locker` where RPC (`badrpc`) errors were not handled correctly, causing false-positive lock successes. This could lead to inconsistent lock states and deadlocks in clustered deployments.
+
+#### Security
+
+
+- [#15159](https://github.com/emqx/emqx/pull/15159) Now, after a CRL Distribution Point URL fails to be refreshed after a number of consecutive attempts (60 s by default), it'll be evicted and stop being refreshed, instead of flooding logs.
```

Comment:
```suggestion
- [#15159](https://github.com/emqx/emqx/pull/15159) Now, after a CRL Distribution Point URL fails to be refreshed after a number of consecutive attempts (60 seconds by default), it'll be evicted and stop being refreshed, instead of flooding logs.
```